include/RAJA/policy/openmp/policy.hpp

```diff
@@ -126,16 +136,48 @@ struct omp_for_schedule_exec : make_policy_pattern_launch_platform_t<Policy::ope
                                                               omp::For,
                                                               Sched> {
     static_assert(std::is_base_of<::RAJA::policy::omp::internal::ScheduleTag, Sched>::value,
-        "Schedule must be one of: Auto|Runtime|Static|Dynamic|Guided");
+        "Schedule type must be one of: Auto|Runtime|Static|Dynamic|Guided");
 };
 
+ 
+///
+///  Internal type aliases supporting 'omp for schedule( )' for specific
+///  schedule types.
+///
 using omp_for_exec = omp_for_schedule_exec<Auto>;
```

Comment:
I'd be tempted to make this `omp_for_schedule_exec<omp::Static<-1>>` personally.

Reply:
That was like that before this PR. The 'Auto' type refers to the non-schedule implementation. So,
omp_for_exec --> #pragma omp for.  I think it would be confusing to make this change. To get what you suggest, it's more consistent with what we have to use omp_for_static_exec. 
I added the extra aliases because I found the nested namespaces too verbose and confusing for the scheduling specification.